pkgs/development/python-modules/doorbirdpy/default.nix

```diff
@@ -0,0 +1,32 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, requests
+}:
+
+buildPythonPackage rec {
+  pname = "doorbirdpy";
+  version = "2.1.0";
+
+  src = fetchPypi {
+    pname = "DoorBirdPy";
+    inherit version;
+    sha256 = "ed0e94953cdf96111c7f73c5fcf358f65dc0ff5e47f63fc057bf18ca7512e606";
+  };
+
+  propagatedBuildInputs = [
+    requests
+  ];
+
+  # no tests on PyPI, no tags on GitLab
```

Comment:
Let's hope that the tests get added to the PyPI release or the source is tagged.

https://gitlab.com/klikini/doorbirdpy/-/issues/3

Reply:
technically, this just makes it as much of a guarantee as a normal pypi install :)